src/main/java/data/Account.java

```diff
@@ -196,4 +196,20 @@ public String buildJsonConfig() {
       getCurrency(), getDirection(), getEntity(), getMatchingMode(), 
       getAccountID(), getAggregationMode());
   }
+
+  /**
+   * @param request a request containing form data to edit existing account
```

Comment:
Maybe add a colon after the first request. I had to read the comment a few times to know what you were trying to say here.